src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,34 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user?: User;
+};
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo: Todo) => {
+        if (!todo.user) {
+          return null;
+        }
+
+        return <TodoInfo todo={todo} key={todo.id} />;
```

Comment:
```suggestion
        return <TodoInfo todo={todo} key={todo.id} />;
```
it's strange, if todo doesn't have the user, then do not render the todo info at all?